packages/astro/src/core/session.ts

```diff
@@ -431,3 +433,16 @@ export class AstroSession<TDriver extends SessionDriverName = any> {
 		}
 	}
 }
+
+export function resolveSessionDriver(driver: string | undefined): string | null {
+	if (!driver) {
+		return null;
+	}
+	if (driver === 'fs') {
+		return import.meta.resolve(builtinDrivers.fsLite);
+	}
+	if (driver in builtinDrivers) {
+		return import.meta.resolve(builtinDrivers[driver as keyof typeof builtinDrivers]);
+	}
```

Comment:
We support older node versions that may have import.meta.resolve return promises, so it might be safer to also await here. https://nodejs.org/api/esm.html#importmetaresolvespecifier